modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,207 @@
+      ipv6:
+        enabled: false
+    - name: enp3s0
+      type: interface
+      state: down
+      ipv4:
+        enabled: false
+      ipv6:
+        enabled: false
+# ...
+----
+<1> The interface that pulls in the Ignition config file for the bond interface.
+
+. Go to the `./<installation_directory>/auth` directory and modify the `master.ign` and `worker.ign` files and add a `balance-slb` configuration to these files.
+endif::postinstall-bare-metal-ipi[]
+
+ifdef::postinstall-bare-metal-ipi[]
+. Create a `pull-secret.txt` configuration file and add a configuration similar to the following example to the file:
++
+[source,yaml]
+----
+networkConfig: &BOND
+  interfaces:
+    - name: enp1s0
+      type: interface
+      state: up
+      ipv4:
+        enabled: true
+        dhcp: true
+      ipv6:
+        enabled: false
+    - name: enp2s0
+      type: interface
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        enabled: false
+    - name: enp3s0
+      type: interface
+      state: down
+      ipv4:
+        enabled: false
+      ipv6:
+        enabled: false
+# ...
+----
+
+. Specify the path to your `pull-secret.txt` in the `pullSecret` parameter of your NNCP or NAD file.
+endif::postinstall-bare-metal-ipi[]
+
+. Open your NMState configuration file in your editor and add the following configuration in the file:
++
+[source,yaml]
+----
+# ...
+interfaces:
+# ...
+- name: br-phy <1>
+  type: ovs-bridge
+  state: up
+  ipv4:
+    enabled: false
+    dhcp: false
+  ipv6:
+    enabled: false
+    dhcp: false
+  bridge:
+    allow-extra-patch-ports: true
+    port:
+    - name: patch-phy-to-ex
+    - name: ovs-bond
+      link-aggregation:
+        mode: balance-slb <2>
+        port:
+        - name: eno1
+        - name: eno2
+- name: patch-ex-to-phy <3>
+  type: ovs-interface
+  state: up
+  patch:
+    peer: patch-phy-to-ex
+- name: patch-phy-to-ex <4>
+  type: ovs-interface
+  state: up
+  patch:
+    peer: patch-ex-to-phy
+- name: eno1 <5>
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: false
+  ipv6:
+    enabled: false
+- name: eno2 <5>
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: false
+  ipv6:
+    enabled: false
+# ...
+----
+<1> Specify an additional bridge, such as `br-phy`, that isolates the `br-ex` bond interface from any changes to the original network rules that OVN-Kubernetes might make to the network environment.
+<2> Enables `balance-slb` mode for the OVS bridge interface.
+<3> Specify the OVS virtual interface that uses an uplink switch to connect the additional bridge with the br-ex bridge.
+<4> Specify the OVS virtual interface that uses an uplink switch to connect the br-ex bridge with the additional bridge.
+<5> Specify the physical interfaces that exist on your primary network.
+
+. For static IP address assignment, manually reboot each node that exists in your cluster. For DHCP address assignment, you can reboot all nodes in one reboot operation.
```

Comment:
There shouldn't be any additional reboots involved here. I'm guessing this is another qurik of the customer configuration method.

Reply:
Removed:

> For static IP address assignment, manually reboot each node that exists in your cluster. For DHCP address assignment, you can reboot all nodes in one reboot operation.

Reply:
Also removed:

> . Optional: If after the reboot operation and you waited a few minutes, a node or some nodes do not have the `READY` status, enter the following commands:
+
.. Check the current state of an Open vSwitch (OVS) configuration, especially details about bridges, ports, and interfaces, by entering the following command:
+
[source,terminal]
----
$ ovs-vsctl show
----
+ 
.. If you identify the `Bridge br-ex` entry in the output, delete the entry by running the following command:
+
[source,terminal]
----
$ ovs-vsctl del-br br-ex
----